src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -118,7 +118,8 @@
         -->
         <PhysicalLocation Condition="'%(_ResolvedPackageReference.PhysicalLocation)'==''">$([System.String]::new('$(NuGetPackageRoot)%(PackageReference.Identity)/%(PackageReference.Version)').ToLower())</PhysicalLocation>
 
-        <DacpacFile>%(_ResolvedPackageReference.PhysicalLocation)/tools/%(Identity).dacpac</DacpacFile>
+        <DacpacFile>%(_ResolvedPackageReference.PhysicalLocation)/tools/%(PackageReference.Identity).dacpac</DacpacFile>
+        <DacpacFile Condition="Exists('%(_ResolvedPackageReference.PhysicalLocation)/tools/%(PackageReference.DacpacName).dacpac')">%(_ResolvedPackageReference.PhysicalLocation)/tools/%(PackageReference.DacpacName).dacpac</DacpacFile>
```

Comment:
Is there any potential issue with having the `DacpacFile` element appear twice? I know we've said that if the file doesn't exist, it just gets silently ignored. What if a user did something like `<PackageReference Include="MyPackage" Version="160.0.0" DacpacName="MyPackage" />`? In this case, we'd have two equivalent `DacpacFile` elements, would that break anything?

Reply:
I think in that case it will just overwrite it, so it shouldn't be a problem. That being said it is probably a good idea to test it ;). I'll make sure we have that covered in one of the test projects somewhere.

Reply:
I've modified the `TestProjectWithPackageReference` test project to have a `PackageReference` to `TestProject` with `DacpacName=TestProject` and that works like a charm.